pkg/hive/hive.go

```diff
@@ -112,7 +113,8 @@ func (s *Service) sendPeers(ctx context.Context, peer swarm.Address, peers []swa
 		if err != nil {
 			_ = stream.Reset()
 		} else {
-			_ = stream.FullClose()
+			time.Sleep(time.Millisecond * 100)
```

Comment:
would be good to have a comment why this is needed, also, it should be in a separate goroutine as discussed

Reply:
the goroutine, for reasons I can't explain, makes this extremely flaky